conans/test/functional/configuration/profile_test.py

```diff
@@ -694,7 +694,7 @@ def configure(self):
     assert "I'm None and my shared is False" in client.out
 
     # Now the dependency by name
-    client.run("install . -s dep:build_type=Debug -o dep:shared=True")
+    client.run("install . -s dep/*:build_type=Debug -o dep*:shared=True")
```

Comment:
```suggestion
    client.run("install . -s dep/*:build_type=Debug -o dep/*:shared=True")
```

I guess it worked because there were no ``dep_other`` package? Was this intended?